thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -209,25 +211,25 @@ export default Ember.Controller.extend({
    * @return {undefined}
    */
   triggerGraphFromMetric(metricId) {
+    const id = metricId.id;
     const maxTime = this.get('maxTime');
-    const filters = this.get('filters');
     const currentEnd = moment(maxTime).isValid()
       ? moment(maxTime).valueOf()
       : moment().subtract(1, 'day').endOf('day').valueOf();
-    const granularity = this.get('selectedGranularity') || this.get('granularities.firstObject');
+    const filters = this.get('selectedFilters') || '';
```

Comment:
fyi we can also use `getWithDefault` here. https://www.emberjs.com/api/ember/2.14/classes/Ember.Object/methods/getWithDefault?anchor=getWithDefault